### PR TITLE
react-react1-week1

### PR DESCRIPTION
**Pull request by 27ManishaPatel**

### CODE REVIEW

File: react/react1/week1/to-do-app/src/App.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
```
Comment (annamourasobral) [inline react/react1/week1/to-do-app/src/App.js:6]: I believe you forgot to import ToDo here.
```suggestion
    <div className="App"> <ToDo />
```
by the way, take care with over nesting of components.

File: react/react1/week1/to-do-app/src/Main.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+export default function Main() {
+    return (<main className="to-do-content">
+        <ul>
+            {ToDoLists.map(list => <ToDoListItems list={list} key={list.id}/>)}
+        </ul>
+    </main>
+    )
+};
+
+function ToDoListItems ({list}) {
```
Comment (annamourasobral) [inline react/react1/week1/to-do-app/src/Main.jsx:29]: this would be more organized in another file, exported as a component.

**Review by annamourasobral — CHANGES_REQUESTED** [review]
Hey, @27ManishaPatel 
Your app worked when I imported the proper files, that happens when you create a big tree of components, you get lost, so take care!
Other than that your code is really nice, congrats!
Happy coding! 👩🏻‍💻

File: react/react1/week1/to-do-app/src/App.js (hunk 1 above)
Comment (27ManishaPatel) [reply react/react1/week1/to-do-app/src/App.js:6]: yes actually  I have exported the components to index.js file from page.jsx

Comment (27ManishaPatel) [reply react/react1/week1/to-do-app/src/App.js:6]: But yes I will take a care thanks😊

File: react/react1/week1/to-do-app/src/ToDo.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import Header from './Header';
+import Footer from './Footer';
+import Main from './Main';
+import './ToDo.css'
+
+export default function ToDo() {
+    return <div className='wrapper'>
+        <Header />
```
Comment (andrii1) [inline react/react1/week1/to-do-app/src/ToDo.jsx:9]: Just a suggestion, I would organize components a bit different - like ToDo, ToDoItem - Header, Main, Footer seems too general.

File: react/react1/week1/to-do-app/src/index.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+import  ToDo  from './ToDo';
+import reportWebVitals from './reportWebVitals';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <ToDo />
+  </React.StrictMode>
+);
+
```
Comment (andrii1) [inline react/react1/week1/to-do-app/src/index.js:13]: Maybe I would add a separate /components folder and keep some components there

File: react/react1/week1/to-do-app/src/ToDo.css [hunk 5]
```diff
@@ -0,0 +1,31 @@
+body {
```
Comment (andrii1) [inline react/react1/week1/to-do-app/src/ToDo.css:1]: also, you can put ToDo.css in a separate /components folder. As I understand, ideally each bigger component should have its own folder and own css - e.g. /ToDoComponent/ToDo.jsx & /ToDoComponent/ToDo.css - then it is more organized. Not needed now - it's not too big :)

**Pull request merged**